jobs/redis-blacksmith-plans/templates/plans/cluster-6/credentials.yml

```diff
@@ -2,7 +2,17 @@
 credentials:
   host:  (( grab jobs.node/0.ips[0] ))
   hosts: (( grab jobs.node.ips ))
-  port:  6379
+<% if p("redis.tls.dual-mode") -%>
```

Comment:
Where tls and tls.dual-mode are both not enabled, the port is 0 and the tls port is also undefined.